scripts/zones/Selbina/Zone.lua

```diff
@@ -7,12 +7,19 @@ local zoneObject = {}
 
 zoneObject.onInitialize = function(zone)
     xi.server.setExplorerMoogles(ID.npc.EXPLORER_MOOGLE)
+    InitializeFishingContestSystem()
 end
 
 zoneObject.onGameHour = function(zone)
     SetServerVariable('Selbina_Destination', math.random(1, 100))
 end
 
+zoneObject.onZoneTick = function(zone)
```

Comment:
Yes -- The tick is only checking to see if the stage progression should trigger (and then progressing if it should).  Once someone is in the zone, it will progress.